pkg/kubectl/cmd/cmd.go

```diff
@@ -126,7 +126,9 @@ __kubectl_parse_config()
 {
     local template kubectl_out
     template="{{ range .$1  }}{{ .name }} {{ end }}"
-    if kubectl_out=$(kubectl config $(__kubectl_override_flags) -o template --template="${template}" view 2>/dev/null); then
+    completion_cmd="kubectl config $(__kubectl_override_flags) -o template --template=\"${template}\" view"
```

Comment:
Could we have this into a new function, like `__kubectl_debug_out`?

Reply:
Let me try!

Reply:
Looks great now. Awesome!